steps/tuist/0.2.1/step.yml

```diff
@@ -0,0 +1,42 @@
+title: tuist
```

Comment:
NOTE: `Run Tuist` was better in line with the Step title requirements: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title

Please consider updating the title in the next release.